core/tests/webdriverio/accessibility.js

```diff
@@ -481,7 +481,7 @@ describe('screenreader and keyboard user accessibility features', function () {
     await explorationPlayerPage.expectExplorationToNotBeOver();
 
     // Should test the skip to main content shortcut.
-    var mainContent = $('.e2e-test-main-content');
+    var mainContent = await $('.e2e-test-main-content');
```

Comment:
@jayam04 do we need `await` with all selectors in webdriverio tests? It looks like it from the [docs](https://webdriver.io/docs/api/element/$), but I don't remember us having to do this before.